docs/js/in-open-shadow-dom.js

```diff
@@ -29,20 +50,50 @@ module.exports = () => {
       shadowEl.appendChild(styleLinkEl);
       shadowEl.appendChild(modalEl);
 
-      const focusTrap = createFocusTrap(modalEl, {
-        onActivate: () => modalEl.classList.add('is-active'),
-        onDeactivate: () => modalEl.classList.remove('is-active'),
-        escapeDeactivates: true,
+      let clickOutsideDeactivates = false;
+
+      const initialize = function () {
+        return createFocusTrap(modalEl, {
+          onActivate: () => modalEl.classList.add('is-active'),
+          onDeactivate: () => modalEl.classList.remove('is-active'),
+          clickOutsideDeactivates,
+          escapeDeactivates: true,
+          tabbableOptions: {
+            getShadowRoot: true,
+          },
+        });
+      };
+
+      let focusTrap = initialize();
+
+      const checkbox = document.getElementById(
+        'checkbox-clickoutsidedeactivates-in-open-shadow'
+      );
+
+      checkbox.addEventListener('change', function () {
+        clickOutsideDeactivates = checkbox.checked;
+        focusTrap.deactivate();
+        focusTrap = initialize();
       });
```

Comment:
The addition of the checkbox to control the outside click behavior is nice, but I don't think it was totally necessary. We could've just kept the demo with that setting added in, and a comment next to the setting explaining why it was important to have that setting turned on. With the checkbox there now, I feel like it becomes important to describe, in the demo, its reason for being there (especially since we have to manually test this one too) -- that being that even though an outside click will disable, clicking particularly on the "shadow span" inside the trap will not be considered outside the trap, and thus the trap will remain active.